Makefile

```diff
@@ -121,3 +138,10 @@ GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
 rm -rf $$TMP_DIR ;\
 }
 endef
+
+# Run all tests
+.PHONY: e2e-test
+e2e-test: 
+	# pull nginx image locally
+	docker pull nginx
```

Comment:
i don't think we need since this will pull to agent but not cluster

Reply:
yes was for testing, will remove it.